core/server/data/migrations/versions/3.25/01-add-members-stripe-webhook-settings.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+    config: {
+        transaction: true
+    },
+
+    async up({transacting: knex}) {
+        await knex('settings')
+            .update({
+                group: 'core',
+                type: 'string',
```

Comment:
I figured it's best to be as explicit as possible here? But can remove `type` and `flags` if people think it's terbil

Reply:
_n.b._ technically this doesn't need a migration at all, because they are core settings being added, but I think for the sake of consistency we should include it

Reply:
Think explicit migration is good idea, and tho we probably won't happen but also guards against if we ever change the default group value from `core` to anything else 👍 

IMO the `type` and `flags` can be left out mostly because thats what we have been doing so far, but NABD either ways 🤷